cpp/src/kmeans/kmeans_mg_impl.cuh

```diff
@@ -395,6 +395,7 @@ void initKMeansPlusPlus(const raft::handle_t& handle,
     auto weight_view =
       raft::make_device_vector_view<const DataT, IndexT>(weight.data_handle(), weight.extent(0));
     raft::cluster::kmeans::KMeansParams default_params;
+    default_params.verbosity  = params.verbosity;
```

Comment:
@lowener, it's been awhile since I've looked through the OPG k-means algorithm at length but I'm curious why we are constructing this `default_params` instance from scratch and not just copying the kmeans params which was passed in and changing the values as needed.